packages/ui/src/pages/PipeErrorHandler/PipeErrorHandlerPage.tsx

```diff
@@ -0,0 +1,56 @@
+import { TextContent, Title } from '@patternfly/react-core';
+import { FunctionComponent, useCallback, useContext, useMemo } from 'react';
+import { MetadataEditor } from '../../components/MetadataEditor';
+import { EntitiesContext } from '../../providers/entities.provider';
+import { PipeErrorHandler as PipeErrorHandlerType } from '@kaoto-next/camel-catalog/types';
+import { PipeResource, SourceSchemaType } from '../../models/camel';
+import { useSchemasStore } from '../../store';
+
+export const PipeErrorHandlerPage: FunctionComponent = () => {
+  const schemaMap = useSchemasStore((state) => state.schemas);
+  const entitiesContext = useContext(EntitiesContext);
+  const pipeResource = entitiesContext?.camelResource as PipeResource;
+
+  const errorHandlerSchema = useMemo(() => {
+    return schemaMap['PipeErrorHandler'].schema;
+  }, [schemaMap]);
+
+  const isSupported = useMemo(() => {
+    return pipeResource && pipeResource.getType() in [SourceSchemaType.Pipe, SourceSchemaType.KameletBinding];
```

Comment:
[curious-question] Is this working as expected?

The `in` operator looks for keys in objects, but in this case, we're looking for strings in a number-keyed object (Array)

I guess this should be instead:
```suggestion
    return pipeResource && [SourceSchemaType.Pipe, SourceSchemaType.KameletBinding].includes(pipeResource.getType());
```

Reply:
ahhh right, it doesn't work for array while works for enum...
```typescript
const array = ['strA', 'strB', 'strC'];

console.log('strB' in array);  // <-- false

enum Types {
    TypeA, TypeB, TypeC
};

console.log('TypeA' in Types); // <--- true
```